src/main/java/pl/stormit/eduquiz/quizcreator/domain/quiz/Quiz.java

```diff
@@ -38,8 +43,10 @@ public class Quiz {
     private User user;
 
     @OneToMany(mappedBy = "quiz", fetch = FetchType.EAGER)
+    @NotEmpty
     private List<Question> questions;
 
     @OneToMany(mappedBy = "quiz", fetch = FetchType.EAGER)
+    @NotEmpty
```

Comment:
I don't know if there will be an exception here:
We have created a new quiz, but we don't have a rare game played yet.